### ISSUE TITLE
Addition of RxLibraries

### ISSUE THREAD

**Issue by pavlospt** [zsoltk/RxAndroidLibs#15]
https://github.com/pavlospt/RxFile
https://github.com/pavlospt/RxIAPv3

**Comment by zsoltk** [comment]
@pavlospt thanks for the input! I guess the reason you opened an issue instead of a PR is to discuss them?

RxFile: nice, with some observations (method signature error in README (most likely a copy-paste error), separation of concerns (file & bitmap), using android logger with no option to disable it, using Log.e everywhere even for debug level messages). Would you like me to open issues for them and discuss them there?

RxIAPv3: a great idea, but since it's clearly stated it's an alpha version, are you sure it's ready to merge? 

Both of them: a more organized wiki with crystal clear usage examples would be really nice.

Basically I'd be glad to merge them, though some improvements would be desired.

**Comment by pavlospt** [comment]
Hello thank you for your great feedback.  I will try to apply it to both libraries! 

RxIAPv3, it is stated that it is in Alpha version mainly because it is a rewrite of a wrapping into Rx. It is in my plans to implement it in one of my apps along with the RxFile library. 

Do you think that i should implement a way to turn off the debug messages in the RxFile library, since there are no loggable sensitive information taking place. I use Log.e mainly because i log only on the error level :P
